### ISSUE TITLE
Linchpin doesn't work with Ansible 2.5

### ISSUE THREAD

**Issue by lukas-bednar** [CentOS-PaaS-SIG/linchpin#516]
With new `ansible-2.5.0` release, linchpin is failing on following issue:
```
$ linchpin -w linchpin-ws/ --creds-path /home/my-qe-jenkins/.config/openstack -d Data.my_cluster.yml up my_cluster
Traceback (most recent call last):
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/bin/linchpin", line 11, in <module>
    sys.exit(runcli())
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/linchpin/shell/__init__.py", line 181, in up
    return_code, results = lpcli.lp_up(targets=targets, run_id=run_id)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/linchpin/cli/__init__.py", line 204, in lp_up
    run_id=run_id)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/linchpin/cli/__init__.py", line 362, in _execute
    return self.do_action(prov_data, action=action, run_id=run_id)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/linchpin/__init__.py", line 566, in do_action
    console=ansible_console)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/linchpin/__init__.py", line 642, in _invoke_playbooks
    console=console)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/linchpin/ansible_runner.py", line 171, in ansible_runner
    cb = PlaybookCallback(options=options)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/linchpin/callbacks.py", line 11, in __init__
    options=options)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/ansible/plugins/callback/__init__.py", line 78, in __init__
    self.set_options(options)
  File "/home/my-qe-jenkins/jenkins/workspace/deploy-my-cluster/.venv/lib/python2.7/site-packages/ansible/plugins/callback/__init__.py", line 97, in set_options
    self._plugin_options = C.config.get_plugin_options(get_plugin_class(self), self._load_name, keys=task_keys, variables=var_options)
AttributeError: 'PlaybookCallback' object has no attribute '_load_name'
```

For meantime I would put `ansible<2.5` restriction into `requirements.txt`.

**Comment by herlo** [comment]
@lukas-bednar I have put in a fix to the requirements.txt which doesn't allow ansible 2.5.0 for now. I'll leave this issue open and address the updated ansible release as soon as possible.

**Comment by herlo** [comment]
This has been addressed with v1.5.2 for now.  Targeting v1.6.0 to enable ansible 2.5.0+ (it could be sooner, however).

**Comment by samvarankashyap** [comment]
@herlo Just found! [ansible-ruuner](https://github.com/ansible/ansible-runner) you might want to consider to replace our custom ansible-runner in near future. It might resolve this issue if implemented.

**Comment by herlo** [comment]
@samvarankashyap that looks interesting. Not sure we want to have another dependency, but we could maybe find something that works with it that's stable. Good suggestion.

**Comment by samvarankashyap** [comment]
fixed by #564
